tests/conda_package_testfiles/test_conda_bear/git/config

```diff
@@ -0,0 +1,11 @@
+[core]
+	repositoryformatversion = 0
+	filemode = true
+	bare = false
+	logallrefupdates = true
+[remote "origin"]
+	url = ../test_conda_bear_remote/
+	fetch = +refs/heads/*:refs/remotes/origin/*
+[branch "master"]
+	remote = origin
+	merge = refs/heads/master
```

Comment:
The `config` could literally have just the URL but I wrote a big one for the sake of making it look real.